ui/app/clinical/index.html

```diff
@@ -561,6 +561,7 @@ <h3 class="close" ng-click="closeTeleConsultation()">
         <script src="common/services/treatmentService.js"></script>
         <script src="common/services/allergyService.js"></script>
         <script src="common/models/auditLogEventDetails.js"></script>
+        <script src="../common/services/allergyService.js"></script>
```

Comment:
What's the need of this file?

Reply:
since we moved the allergyService to the common directory, we need to include the script to be accessed in the clinical module